README.md

```diff
@@ -95,12 +95,12 @@ Monero uses a fixed-schedule hard fork mechanism to implement new features. This
 Dates are provided in the format YYYY-MM-DD. 
 
 
-| Fork Date              | Consensus version | Minimum Monero Version | Recommended Monero Version | Details            |  
+| Fork Date         | Consensus version | Minimum Monero Version | Recommended Monero Version | Details |  
 | ----------------- | ----------------- | ---------------------- | -------------------------- | ------------------ |
-| 2016-09-21        | v3                | v0.9.4                 | v0.10.0                    | Splits coinbase into denominations  |
-| 2017-01-05        | v4                | v0.10.1                 | v0.10.2.1                   | Allow normal and RingCT transactions |
-| 2017-04-15        | v5                | v0.10.3.0               | v0.10.3.1                    | Adjusted minimum blocksize and fee algorithm      |
-| 2017-09-21        | v6                | Not determined as of 2017-03-27                | Not determined as of 2017-03-27                    | Allow only RingCT transactions      |
+| 2016-09-21        | v3                | v0.9.4                 | v0.10.0                    | Splits coinbase into denominations |
+| 2017-01-05        | v4                | v0.10.1                | v0.10.2.1                  | Allow normal and RingCT transactions |
+| 2017-04-15        | v5                | v0.10.3.0              | v0.10.3.1                  | Adjusted minimum blocksize and fee algorithm |
+| 2017-09-21        | v6                | v0.11.0.0              | v0.11.0.0                  | Allow only RingCT transactions, default ring size is 5  |
```

Comment:
The wallet default was already 5. This hardfork changes the minimum to 5.

Reply:
That's what I meant, but expressed badly. Fixed.